modules/systemd-journal/systemd-journal-grammar.ym

```diff
@@ -101,7 +101,7 @@ source_systemd_journal_option
     }
   | KW_TIME_ZONE '(' string ')'
     {
-      if (last_journal_reader_options->recv_time_zone);
```

Comment:
It was just the propagation of flags and some other configuration fixes that brought this up via a warning.